File: ServicoSpring/src/main/java/com/ufrpe/dc/pisd/crimes/beans/Crime.java
package com.ufrpe.dc.pisd.crimes.beans;

import lombok.Data;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import java.io.Serializable;
import java.util.Date;

@Entity
@Data
@Table(name = "crime")
public class Crime implements Serializable {

    @Id
    @GeneratedValue
    private Long crime_id;
    private String cidade;

    @Temporal(TemporalType.DATE)
    @NotNull
    private Date data_ocorrencia;
    private Integer idade;

  private String nome;

  private String objeto_utilizado;

  private Integer qtd_vitimas;

  private String sexo;

  private String tipo_crime;
}

File: ServicoExpress/src/model/Crime.js

module.exports = (sequelize, Sequelize) =>{
    return sequelize.define('crime',{
        'crime_id': {type: Sequelize.INTEGER,autoIncrement: true, primaryKey: true},
        'cidade': {type: Sequelize.STRING},
        'data_ocorrencia': {type: Sequelize.DATE},
        'idade': {type: Sequelize.INTEGER},
        'nome': {type: Sequelize.STRING},
        'objeto_utulizado': {type: Sequelize.STRING},
        'qtd_vitimas': {type: Sequelize.STRING},
        'sexo': {type: Sequelize.STRING},
        'tipo_crime': {type: Sequelize.STRING}
    },{
        timestamps: false
    });
}